src/test/java/com/exasol/bucketfs/native_image/NativeImageTest.java

```diff
@@ -0,0 +1,22 @@
+package com.exasol.bucketfs.native_image;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.startsWith;
+
+import java.io.IOException;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import com.exasol.bucketfs.JarExecutor;
+
+@Tag("native-image")
+class NativeImageTest {
+
+    @Test
+    void testNativeImage() throws IOException, InterruptedException {
+        final JarExecutor executor = new JarExecutor().run("--help");
```

Comment:
This executes the JAR, not the native image!

Reply:
Oh thanks - I need to fix this.